apps/web/src/components/auth/forgot-password/form.tsx

```diff
@@ -0,0 +1,134 @@
+"use client";
+
+import type { HTMLAttributes } from "react";
+import type { z } from "zod";
+import React, { useState } from "react";
+import Link from "next/link";
+import { useRouter } from "next/navigation";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { toast } from "sonner";
+import { Button } from "ui/components/button";
+import { Form, FormField, useForm } from "ui/components/form";
+import { Input } from "ui/components/input";
+import { InputOTP, InputOTPGroup, InputOTPSlot } from "ui/components/input-otp";
+import { PasswordInput } from "ui/components/password-input";
+import { cn } from "ui/lib/utils";
+import {
+  requestPasswordResetSchema,
+  resetPasswordSchema,
+} from "validators/auth";
+
+import { api } from "~/trpc/react";
+
+type Props = HTMLAttributes<HTMLDivElement>;
+type ForgotPasswordFormData = z.infer<typeof resetPasswordSchema>;
+
+export const ForgotPasswordForm = ({ className, ...props }: Props) => {
```

Comment:
use `React.FC` type with arrow function and pass type as a generic

Reply:
why do we need this? it will infer type by default?

Reply:
Change in the plan, we don't need this